js/signaling.js

```diff
@@ -155,27 +162,35 @@
 	/**
 	 * @private
 	 */
-	InternalSignaling.prototype._openEventSource = function() {
+	InternalSignaling.prototype._startPullingMessages = function() {
 		// Connect to the messages endpoint and pull for new messages
-		this.source = new OC.EventSource(OC.generateUrl('/apps/spreed/messages'));
-
-		this.source.listen('usersInRoom', function(users) {
-			this._trigger('usersInRoom', [users]);
-		}.bind(this));
-		this.source.listen('message', function(message) {
-			if (typeof(message) === 'string') {
-				message = JSON.parse(message);
-			}
-			this._trigger('message', [message]);
-		}.bind(this));
-		this.source.listen('__internal__', function(data) {
-			if (data === 'close') {
-				console.log('signaling connection closed - will reopen');
-				setTimeout(function() {
-					this._openEventSource();
-				}.bind(this), 0);
-			}
-		}.bind(this));
+		$.ajax({
```

Comment:
You should also handle errors here and start again if a request failed. Probably should use increasing delays if lots of requests fail in a short time (e.g. if your network connection is interrupted).

Reply:
@Ivansss can you fix this?

Reply:
Fixed! Retry after 5 seconds if pull messages fails.